src/components/Header/Header.jsx

```diff
@@ -1,86 +1,55 @@
 import React from 'react';
-import { useQuery } from 'react-query';
-import { BellIcon } from '@chakra-ui/icons';
-import {
-  Flex,
-  Heading,
-  IconButton,
-  Text,
-  useColorModeValue
-} from '@chakra-ui/react';
-import i18next from 'i18next';
-
-import loginAction from '@/api/AuthProvider';
-import { LogoWalletIcon } from '@/assets';
-import { theme } from '@/theme';
+import { Box, Flex, useColorModeValue } from '@chakra-ui/react';
 
 import ColorModeToggle from '../ColorModeToggle';
 
+import { Logo } from './Logo';
 import { Navbar } from './Navbar';
+import { NavDrawer } from './NavDrawer';
+import { Notification } from './Notification';
 import { UserMenu } from './UserMenu';
+import { UserName } from './UserName';
 
-export const Header = () => {
+export const Header = (isOpen) => {
   const headerBgThemeColor = useColorModeValue('orange.100', 'teal.900');
   const headerTextColor = useColorModeValue('teal.900', 'orange.300');
-  const iconsThemeColor = useColorModeValue('teal.900', 'orange.300');
-  const iconsHoverThemeColor = useColorModeValue('teal.900', 'orange.300');
-
-  const { data } = useQuery(['user'], () => loginAction());
-
   return (
     <Flex
       as="header"
       justify="center"
       minH="100px"
       bg={headerBgThemeColor}
-      boxShadow="base"
+      boxShadow="headerShadow"
+      color={headerTextColor}
     >
       <Flex
         align="center"
         justify="space-between"
-        maxW="1200px"
-        w="100%"
+        maxWidth="1200px"
```

Comment:
```suggestion
        maxW="container.xl"
```

I suggest to use here Chakra's default value for max-width just to be more consistent.
Basically, it contains '1280px' value in it and we already have an access to that variable with Chakra's predefined sizes.

https://chakra-ui.com/docs/styled-system/theme#sizes

And we can use `maxW` shorthand here. I don't understand why you change it)